src/ukify/ukify.py

```diff
@@ -1695,7 +1695,7 @@ def inspect_section(
 
     if ttype == 'text':
         try:
-            struct['text'] = data.decode()
+            struct['text'] = data.decode(errors='replace').replace('\x00', '')
```

Comment:
Maybe better to escape it rather than remove?

Reply:
There's no point in showing null characters here, it's just padding